lib/utils/confirm.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+'use strict'
+
+const inquirer = require('inquirer')
+
+function confirm (
+  force /* : boolean */,
+  env /* : string */
+) /* : Promise<boolean> */ {
+  if (force) {
+    return Promise.resolve(true)
+  }
+  const promptQuestions = [{
+    type: 'confirm',
+    name: 'confirmation',
+    message: `Are you sure you want to deploy to environment "${env}": [Y]`
```

Comment:
should there be a `default: true` property on the object or does putting `[Y]` in the message do that?